### DISCUSSION TITLE
Get the user's geographic location information regularly in ios

### DISCUSSION THREAD

**Discussion by jason-apj** [dotnet/maui#10268] [Q&A]
I have a need to get the user's status and location information regularly when the app enters the background or the mobile phone screen is turned off. I checked the relevant documents of the MAUI and did not see the relevant information. I have no idea about this. I hope someone can help me

**Comment by aritchie** [comment]
There is an alpha version of Shiny.NET that supports this.  You can find it here with samples: https://github.com/shinyorg/shiny 

What you are requesting isn't something MAUI does.  MAUI is for user interface.
